### PR TITLE
tpm.py: add active_pcr_banks element

### PR DESCRIPTION
**Pull request by yanqzhan**
Add active_pcr_banks element to tpm device construction.

Signed-off-by: Yanqiu Zhang <yanqzhan@redhat.com>

### CODE REVIEW

**Comment by yanqzhan** [comment]
Local test pass:
1. when:
for pcrbank in {'sha1', 'sha256', 'sha384', 'sha512'}:
    backend_pcrbank.add_pcrbank(pcrbank)
backend.active_pcr_banks = backend_pcrbank
get:
    <tpm model="tpm-crb">
      <backend type="emulator" version="2.0">
        <encryption secret="aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee" />
        <active_pcr_banks>
          <sha1 />
          <sha256 />
          <sha384 />
          <sha512 />
        </active_pcr_banks>
      </backend>
    </tpm>
2. when:
for pcrbank in {'sha1', 'sha256', 'sha384', 'sha512'}:
    backend_pcrbank.add_pcrbank(pcrbank)
if backend_pcrbank.has_pcrbank('sha1'):
    backend_pcrbank.remove_pcrbank('sha1')
backend.active_pcr_banks = backend_pcrbank
get:
    <tpm model="tpm-crb">
      <backend type="emulator" version="2.0">
        <encryption secret="aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee" />
        <active_pcr_banks>
          <sha256 />
          <sha384 />
          <sha512 />
        </active_pcr_banks>
      </backend>
    </tpm>

Test method:
config in tpm_device.py under line: if secret_uuid == 'nonexist', run this case and check logs.
# avocado run --vt-type libvirt tpm_device.negative_test.emulator.encrypt_secret.nonexist_secret --vt-machine-type q35
 (1/1) type_specific.io-github-autotest-libvirt.virtual_devices.tpm_device.negative_test.emulator.encrypt_secret.nonexist_secret: PASS (7.19 s)

**Comment by yanqzhan** [comment]
local run still pass with latest commit.

**Review by dzhengfy — APPROVED** [review]
LGTM

File: virttest/libvirt_xml/devices/tpm.py [hunk 1]
```diff
@@ -70,5 +77,68 @@ def __init__(self, virsh_instance=base.base.virsh):
...
+                                     subclass=self.ActivePCRBanks,
+                                     subclass_dargs={
+                                         'virsh_instance': virsh_instance})
             super(self.__class__, self).__init__(virsh_instance=virsh_instance)
             self.xml = '<backend/>'
+
+        class ActivePCRBanks(base.base.LibvirtXMLBase):
+            """
+            Tpm active_pcr_banks xml class.
+
```
Comment (chunfuwen) [inline virttest/libvirt_xml/devices/tpm.py:93]: Perhaps you can add what one simple activePCRBanks looks like here?

Comment (yanqzhan) [reply virttest/libvirt_xml/devices/tpm.py:93]: Do you mean xml example like following?
<active_pcr_banks>
<sha256 />
</active_pcr_banks>

Comment (chunfuwen) [reply virttest/libvirt_xml/devices/tpm.py:93]: @yanqzhan  yes

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**